### PR TITLE
[MTM-58448] Extract the Reports view into a separate plugin announcement

### PR DESCRIPTION
**Pull request by jdreSAG**
announcement of potentially breaking change https://cumulocity.atlassian.net/browse/MTM-58448

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58448
+version: 1020.0.0
+---
+In the upcoming version, we will extract Reports module from Cockpit application and we will add is as a separate plugin.
```
Comment (sag-lukjan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:17]: Besides one spelling, maybe:
```suggestion
In the upcoming version we will extract the Reports module from the Cockpit application and add it as a separate plugin.
```
?

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+This is just first step towards removing Reports completely and replace is with Dashboard manager.
```
Comment (sag-lukjan) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:18]:
```suggestion
This is just a first step towards removing Reports completely and replacing it with Dashboard Manager.
```
?

**Review by sag-lukjan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58448
+version: 1020.0.0
+---
+In the upcoming version we will extract the Reports module from the Cockpit application and add it as a separate plugin.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:17]:
```suggestion
In the upcoming version the report module is extracted from the Cockpit application and added as a separate plugin.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+This is just a first step towards removing Reports completely and replacing it with Dashboard Manager.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:18]:
```suggestion
This is a first step towards removing the reports module entirely and replacing it with the dashboard manager.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+It might be breaking change as Reports module won't be part of `@c8y/ngx-components/context-dashboard` anymore, but will be part of `@c8y/ngx-components/report-dashboard`.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:19]:
```suggestion
This change might be a breaking change as the reports module will no longer be part of `@c8y/ngx-components/context-dashboard`, but will be part of `@c8y/ngx-components/report-dashboard`.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+This change won't affect visual and UX of Cockpit application nor add any new functionalities.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:21]:
```suggestion
This change does not affect the layout and UX of the Cockpit application nor does it add any new functionalities.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+As side effect, performance improvement might be noticed as Reports from now on will be loaded lazily (only when user navigates to Reports).
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:22]:
```suggestion
As a side effect, a performance improvement might be noticed as reports from now on will be loaded lazily (that is, only when users navigate to the **Reports** page).
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+---
+date: 2024-04-26
+title: Extract the Reports view into a separate plugin
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md:3]:
```suggestion
title: Reports page extracted into a separate plugin
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**